pydicom/util/codify.py

```diff
@@ -333,8 +341,10 @@ def code_file(filename, exclude_size=None, include_private=False):
 
     # If requested, write a code line to save the dataset
     if args.save_as:
-        save_line = "\nds.save_as('{filename}')".format(filename=args.save_as)
+        msg = "\nds.save_as('{filename}', write_like_original=False)"
+        save_line = msg.format(filename=args.save_as)
```

Comment:
better make `'{filename}'` unicode (e.g. `u'{filename}'`), in case the file name is not ASCII